From 6bae7d5949dacd300ccbf31491a005085ef7dc03 Mon Sep 17 00:00:00 2001
From: Ryan QIAN <b32804@freescale.com>
Date: Fri, 10 Feb 2012 16:02:11 +0800
Subject: [PATCH 1004/2463] ENGR00174224 [MX6Q]: Add new AR6003 driver to
 3.0.15 into default config

- Add cfg80211, Atheros Wifi driver into default kernel config.

Signed-off-by: Ryan QIAN <b32804@freescale.com>
---
 arch/arm/configs/imx6_defconfig |   16 ++++++++++++----
 1 files changed, 12 insertions(+), 4 deletions(-)

diff --git a/arch/arm/configs/imx6_defconfig b/arch/arm/configs/imx6_defconfig
index 9a82058..d017ed1 100644
--- a/arch/arm/configs/imx6_defconfig
+++ b/arch/arm/configs/imx6_defconfig
@@ -637,17 +637,17 @@ CONFIG_WEXT_CORE=y
 CONFIG_WEXT_PROC=y
 CONFIG_WEXT_SPY=y
 CONFIG_WEXT_PRIV=y
-# CONFIG_CFG80211 is not set
+CONFIG_CFG80211=y
 CONFIG_WIRELESS_EXT_SYSFS=y
+CONFIG_CFG80211_DEFAULT_PS=y
+CONFIG_CFG80211_WEXT=y
 CONFIG_LIB80211=y
 CONFIG_LIB80211_CRYPT_WEP=y
 CONFIG_LIB80211_CRYPT_CCMP=y
 CONFIG_LIB80211_CRYPT_TKIP=y
 # CONFIG_LIB80211_DEBUG is not set
 
-#
-# CFG80211 needs to be enabled for MAC80211
-#
+# CONFIG_MAC80211 is not set
 # CONFIG_WIMAX is not set
 CONFIG_RFKILL=y
 CONFIG_RFKILL_INPUT=y
@@ -953,8 +953,16 @@ CONFIG_FEC=y
 # CONFIG_NETDEV_10000 is not set
 CONFIG_WLAN=y
 # CONFIG_USB_ZD1201 is not set
+# CONFIG_USB_NET_RNDIS_WLAN is not set
+CONFIG_ATH_COMMON=m
+# CONFIG_ATH_DEBUG is not set
+CONFIG_ATH6KL=m
+# CONFIG_ATH6KL_DEBUG is not set
 CONFIG_HOSTAP=y
 # CONFIG_HOSTAP_FIRMWARE is not set
+# CONFIG_IWM is not set
+# CONFIG_LIBERTAS is not set
+# CONFIG_MWIFIEX is not set
 
 #
 # Enable WiMAX (Networking options) to see the WiMAX drivers
-- 
1.7.7.4

